src/gridfs/upload.ts

```diff
@@ -1,14 +1,20 @@
 import { Writable } from 'stream';
-import { MongoError, AnyError, MONGODB_ERROR_CODES, MongoDriverError } from '../error';
-import { WriteConcern } from './../write_concern';
-import { PromiseProvider } from '../promise_provider';
+import type { Document } from '../bson';
 import { ObjectId } from '../bson';
+import type { Collection } from '../collection';
+import {
+  AnyError,
+  MONGODB_ERROR_CODES,
+  MongoDriverError,
+  MongoError,
+  MongoStreamClosedError
+} from '../error';
+import { PromiseProvider } from '../promise_provider';
 import type { Callback } from '../utils';
-import type { Document } from '../bson';
-import type { GridFSBucket } from './index';
-import type { GridFSFile } from './download';
 import type { WriteConcernOptions } from '../write_concern';
-import type { Collection } from '../collection';
+import { WriteConcern } from './../write_concern';
+import type { GridFSFile } from './download';
+import type { GridFSBucket } from './index';
```

Comment:
In the same vein as before, just curious as to what the intention was with reordering the imports.

Reply:
The almighty ESLint reordered things here too.